packages/design-system/src/components/icon-button-with-menu.tsx

```diff
@@ -0,0 +1,148 @@
+import React from "react";
```

Comment:
Do you think we will ever need this component beyond style panel? If no it should not be in design-system